custom_dc/env.list

```diff
@@ -0,0 +1,34 @@
+# API key for accessing the base Data Commons API.
+DC_API_KEY=
+# API key for accessing Maps and Places APIs for exploration tools.
+MAPS_API_KEY=
+# Use local SQLite as the database.
+USE_SQLITE=true
+# Input directory containing files to be processed.
```

Comment:
Should we move these to the bottom, with the options for enabling NL etc.?

Reply:
is generating NL embeddings optional?
Are there other options?

Reply:
Let's run everything for now.

Reply:
Done